demoapp/www/js/apis.js

```diff
@@ -0,0 +1,35 @@
+$(document).bind('pageinit', function () {
+
+    //The code below wouldn't work for cordova yet.
```

Comment:
Could you please clarify this comment? In case things aren't going to work now, maybe it would be better to disable this page / throw an alert or so?